apps/judicial-system/web/src/components/Table/Table.tsx

```diff
@@ -180,6 +180,8 @@ const Table: FC<TableProps> = (props) => {
     switch (column) {
       case 'defendants':
         return entry.defendants?.[0]?.name ?? ''
+      case 'defendantsPunishmentType':
```

Comment:
This refers to the fields added [here](https://github.com/island-is/island.is/pull/17285/files#r1890553967)